changelog.d/20250620_171028_PL-133160-merge-insecure-unfree-declarations_scriv.md

```diff
@@ -0,0 +1,26 @@
+<!--
+
+A new changelog entry.
+
+Delete placeholder items that do not apply. Empty sections will be removed
+automatically during release.
+
+Leave the XX.XX as is: this is a placeholder and will be automatically filled
+correctly during the release and helps when backporting over multiple platform
+branches.
+
+-->
+
+### Impact
+
+
+### NixOS XX.XX platform
+
+- Introduced option `flyingcircus.permittedInsecurePackages` to allow additional packages marked as insecure.
+
+- Improved error message when trying to use a package marked as insecure or unfree showing FCIO-specific instructions.
```

Comment:
This is https://github.com/flyingcircusio/nixpkgs/pull/1115 actually. Would like to see those merged together.